plugins/bazaar-backend/src/service/router.ts

```diff
@@ -106,7 +149,7 @@ export async function createRouter(
 
   router.put('/projects', async (request, response) => {
     const bazaarProject = request.body;
-
+    await evaluateRequestPermission(request, bazaarUpdatePermission);
```

Comment:
Could we flip this into an express middleware? If we're just checking against the request and not doing any conditional work, I think that makes more sense.